hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/timeabs/TimeSheetDataController.java

```diff
@@ -48,15 +51,32 @@ public void setTimeSheetDao(TimeSheetDao timeSheetDao) {
     }
 
     
+    /**
+     * Gets Time sheets - limited to most recent 80 results
+     * @param modelMap
+     * @return the most recent timesheets, limited to 80
+     */
     @Secured({"ROLE_VIEW_WEB_CLOCK", "ROLE_VIEW_TIME_CLOCK", "ROLE_VIEW_TIME_SHEET"})
     @ResourceMapping("timeSheets")
     public String getTimeSheets(ModelMap modelMap) {
         final String emplid = PrimaryAttributeUtils.getPrimaryId();
 
         final List<TimeSheet> timeSheets = this.timeSheetDao.getTimeSheets(emplid);
-
-        modelMap.addAttribute("report", timeSheets);
         
+        //sorts the list from most recent time sheets to oldest time sheets
+        Collections.sort(timeSheets, new Comparator<TimeSheet>(){
```

Comment:
I'd have implemented an external `TimeSheetComparator` class so as to unit test it, but it'd be a trivial unit test and I accept that some developers prefer this style of code.